api/MediaList.json

```diff
@@ -238,6 +238,45 @@
             "deprecated": false
           }
         }
+      },
+      "toString": {
```

Comment:
Same question as https://github.com/mdn/browser-compat-data/pull/19093/files#r1134742598 - this can go in if that goes in. 

Specifically, we need a clear data guideline on interpreting that a particular IDL entity, in this case `stringifier` should be expanded out to having a `toString()` method on the interface. It would also be good to understand what the behaviour of that method is - i.e. if it doesn't differ from `value` then should it be documented in BCD separately - even if **perhaps** it makes sense to do so in docs.